src/components/FormRadioButton/index.js

```diff
@@ -21,10 +22,23 @@ const FormRadioButton = ({ input, radioValue, text, className, highlightColor })
   </div>
 )
 
+FormRadioButton.propTypes = {
+  input: PropTypes.object,
+  radioValue: PropTypes.string,
+  text: PropTypes.string,
+  className: PropTypes.string,
+  highlightColor: PropTypes.string,
+}
+
```

Comment:
And here: https://github.com/erikras/redux-form/blob/master/src/propTypes.js#L84